src/ConvertESOMTR/mainModule.f90

```diff
@@ -430,6 +430,7 @@ Subroutine readESOdictionary()
 !'check to make sure the entire file wasn't treated as the data dictionary
 If (lineOfESO /= 'End of Data Dictionary') Then
```

Comment:
It doesn't need another line. That routine is just for reading the data dictionary. If it exits the loop before it gets to the 'End of Data Dictionary' line then it reports an error otherwise it just outputs that line in the IP.ESO file.